shared/test/loop_mocha_utils.js

```diff
@@ -201,7 +207,7 @@ var LoopMochaUtils = (function(global, _) {
    *                             to the listeners.
    */
   function publish() {
-    var args = Array.slice(arguments);
+    var args = Array.prototype.slice.call(arguments);
```

Comment:
I know why it's there, I meant a brief comment in the code, since non-expert JS coders might well assume that arguments is an Array.  :-)